testcrate/tests/lse.rs

```diff
@@ -1,6 +1,7 @@
 #![feature(decl_macro)] // so we can use pub(super)
 #![cfg(all(
     any(target_arch = "aarch64", target_arch = "arm64ec"),
```

Comment:
You can also remove arm64ec here since it's windows-only.

Reply:
Removed it here too